qa/L0_server_status/server_status_test.py

```diff
@@ -141,160 +141,155 @@ def test_unknown_model_version(self):
     def test_model_latest_infer(self):
         input_size = 16
         tensor_shape = (1, input_size)
-        platform_name = {
-            'graphdef': 'tensorflow_graphdef',
-            'netdef': 'caffe2_netdef'
-        }
 
-        # There are 3 versions of *_int32_int32_int32 and all
+        # There are 3 versions of graphdef_int32_int32_int32 and all
         # should be available.
-        for platform in ('graphdef', 'netdef'):
```

Comment:
For these tests I think it is ok to remove "netdef" without a replacement, but keep the for loop in case we want to add something back later.